rbac/migration_tool/sharedSystemRolesReplicatedRoleBindings.py

```diff
@@ -165,14 +158,16 @@ def v1_role_to_v2_mapping(
     return frozenset(v2_role_bindings)
 
 
-candidate_system_roles = {}
 custom_roles_created = 0
 
 
-def extract_system_roles(perm_groupings, v1_role, db_role_id, use_mapping_from_db=False):
+def extract_system_roles(perm_groupings: dict[V1resourcedef, list[str]], v1_role: V1role, binding_mapping: BindingMapping | None):
```

Comment:
It looks like this syntax for union types is supported in Python 3.10 + https://peps.python.org/pep-0604/ and [we're currently requiring 3.9](https://github.com/RedHatInsights/insights-rbac/blob/master/Pipfile#L62) 

Maybe this could work prior to an upgrade? https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Looks like that did the trick ty